pyphi/actual.py

```diff
@@ -410,14 +410,22 @@ def find_causal_link(self, direction, mechanism, purviews=False,
         if not purviews:
             max_ria = _null_ac_ria(self.mechanism_state(direction),
                                    direction, mechanism, None)
+            return CausalLink(max_ria)
         else:
```

Comment:
Now that there's a `return` statement in the `if` clause, you don't need the `else`; you can just remove it and dedent that block.